test.js.md

```diff
@@ -0,0 +1,13 @@
+# Snapshot report for `test.js`
+
+The actual snapshot is saved in `test.js.snap`.
+
+Generated by [AVA](https://ava.li).
+
+## runs without crashing
+
+> Snapshot 1
+
+    `🕵️‍♀️  Glow v1.1.0 (flow-bin v0.63.1)␊
```

Comment:
This snapshot is going to break whenever a new version of Glow is released.

Could just just assert that the stdout contains a message?

Reply:
Yeah good call, will do.